standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1244,36 +1255,36 @@ private TableValidWriteIds getValidWriteIdsForTable(Statement stmt, String fullT
     if (rs.next()) {
       validWriteId = rs.getLong(1);
     } else {
-      // All input write ids should be greater than this
-      validWriteId = 0L;
+      // All input write ids should be greater or equal than this
```

Comment:
Can we do all these validations of write id against the allocated write id after the normal flow of allocation of write ids by caller?

Reply:
done